### PR TITLE
add instructions for the Galaxy platform 

### PR DESCRIPTION
**Pull request by sunyi000**

### CODE REVIEW

**Comment by tischi** [comment]
Hi, thanks a lot, I started looking into it.

I am starting to collect here some general Galaxy UX comments:

1. It is a bit irritating that something very simple that should just take a few milliseconds, such as applying a threshold to an image takes several minutes to execute, because one needs to wait for a job to be scheduled. For teaching and workflow development, would it be possible to set up a local instance that would execute faster? 
2. What's the status of the TIFF viewer? I clicked on the 👁️ but it just downloaded the image

**Comment by sunyi000** [comment]
> Hi, thanks a lot, I started looking into it.
> 
> I am starting to collect here some general Galaxy UX comments:
> 
> 1. It is a bit irritating that something very simple that should just take a few milliseconds, such as applying a threshold to an image takes several minutes to execute, because one needs to wait for a job to be scheduled. For teaching and workflow development, would it be possible to set up a local instance that would execute faster?
> 2. What's the status of the TIFF viewer? I clicked on the 👁️ but it just downloaded the image

1. I agree,, the speed depends on how busy the cluster is. But to have a local instance on user laptop, one should just need to clone the Galaxy repo, switch to a specific branch and run a start script. 
2. The eye icon won't open the tiff viewer. For normal tiff, need to click 'Visualize' from the top menu, and select Tiff Viewer. For ome tiff, there should be a hyperlink 'avivator' (if I remember the name correctly)  above the eye icon.

**Comment by tischi** [comment]
> But to have a local instance on user laptop, one should just need to clone the Galaxy repo, ...

Would it be possible to add instructions for this to the [Tool installation](https://neubias.github.io/training-resources/tool_installation/index.html) module?

**Comment by tischi** [comment]
> For normal tiff, need to click 'Visualize' from the top menu, and select Tiff Viewer. 

This requires to register, while running the other things did not need to register, do you know why?

**Comment by sunyi000** [comment]
> > But to have a local instance on user laptop, one should just need to clone the Galaxy repo, ...
> 
> Would it be possible to add instructions for this to the [Tool installation](https://neubias.github.io/training-resources/tool_installation/index.html) module?

Ok. I will add it.

**Comment by sunyi000** [comment]
> > For normal tiff, need to click 'Visualize' from the top menu, and select Tiff Viewer.
> 
> This requires to register, while running the other things did not need to register, do you know why?

not sure. I will check..:)

File: _includes/auto_threshold/auto_threshold_act1_galaxy.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+- Navigate to [Galaxy](https://usergalaxy.eu)
+- Upload an image
+    - In the Tools panel on the left side, click `Upload Data`.
+    - Click the `Paste/Fetch data` button.
+    - Paste the URLs of the images : [xy_8bit__nuclei_without_offset.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xy_8bit__nuclei_without_offset.tif) and [xy_8bit__nuclei_with_offset.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xy_8bit__nuclei_with_offset.tif)
+    - Click the `Start` button and wait for the upload to complete.
+    - Once the upload is finished, click the `Close` button. The image will now be available in your Galaxy history.
+- Apply a threshhold
+  - In the Tools panel on the left side, search `ImageJ2`.
+  - Choose the tool named `Adjust threshold with ImageJ2`, and click on it.
```
[the remark this answers is not in the document]
Comment (sunyi000) [reply _includes/auto_threshold/auto_threshold_act1_galaxy.md]: changed to a different tool

**Comment by tischi** [comment]
Looks great! I will merge it!

**Pull request merged**